core/src/main/java/com/exadel/aem/backpack/core/dto/response/JcrPackageWrapper.java

```diff
@@ -0,0 +1,68 @@
+package com.exadel.aem.backpack.core.dto.response;
+
+import com.google.gson.Gson;
+import org.apache.jackrabbit.vault.packaging.JcrPackage;
+
+/**
+ * Represents the package modification info.
+ */
+public class JcrPackageWrapper {
+
+    private transient JcrPackage jcrPackage;
+    private String message;
+    private int statusCode;
+
+    public void setJcrPackage(final JcrPackage jcrPackage) {
```

Comment:
Please add minimal docs for setters and getters.

Reply:
done